Engineering/ThreeJS/Cameras in ThreeJS.md

```diff
@@ -0,0 +1,71 @@
+---
+tags: engineering/threejs, threejs, camera
+author: Nguyen Dinh Nam
+date: 2022-09-15
+---
+
+ThreeJS is a JavaScript 3D library that allows developers to develop and describe data in 3 dimensions, and then convert them into 2 dimensions and display them on HTML Canvas. **Camera** is one of the core elements of a ThreeJS project, beside Scene, Renderer and 3D objects.
+
+This article gives basic information about characteristics of the frequently used types of camera in ThreeJS.
+
+## Mostly used types of camera in ThreeJS
+
+You cannot use the `Camera` class directly to initialize a new camera in ThreeJS. Instead, use the `PerspectiveCamera` or `OrthographicCamera` specifically. The perspective camera gives information about depth, distances, etc. since it represents perspective in real life. On the other hand, with an orthographic camera, all sizes are projected the same way and we can easily compare them accurately.
+
+The image below shows the object viewed in perspective camera (upper) and in orthographic camera (lower).
+
+![](https://i.imgur.com/Va74b4E.png)
+
+Place them into the Oxyz coordinate...
+
+![](https://i.imgur.com/uIocTAX.png)
+
+### Perspective camera
+
+![](https://i.imgur.com/OIpfGw7.png)
+
+Some crucial attributes:
+- **fov** (field of view): Camera frustum vertical field of view, i.e. the angle of the view in the yOz plane, in degree. The lower the number, the narrower the view.
```

Comment:
Would be great if we have an illustration for fov in 3D space. Understanding fov in 3D context could be difficult.